### PR TITLE
#1620: display users' level on the users table

### PR DESCRIPTION
**Pull request by sudo-may**

### CODE REVIEW

File: dashboard/src/components/users/UsersTable.vue [hunk 1]
```diff
@@ -60,17 +60,26 @@ limitations under the License.
...
-          <div class="row" :data-cy="`usr_progress-${data.item.userId}`">
-            <div class="col-auto">
-              <span class="font-weight-bold text-primary" data-cy="progressPercent">{{ calcPercent(data.value) }}%</span>
+          <div :data-cy="`usr_progress-${data.item.userId}`">
+            <div class="row">
+              <div class="col-auto">
+                <span class="font-weight-bold text-primary" data-cy="progressPercent">{{ calcPercent(data.value) }}%</span>
+              </div>
+              <div class="col text-right">
+                <span class="text-primary font-weight-bold" data-cy="progressCurrentPoints">{{ data.value | number }}</span> / <span class="font-italic" data-cy="progressTotalPoints">{{ totalPoints | number }}</span>
```
Comment (mosefer) [inline dashboard/src/components/users/UsersTable.vue:69]: The xx / xxxxx points could use a better aria-label for screenreader users

File: dashboard/src/components/users/UsersTable.vue [hunk 2]
```diff
@@ -60,17 +60,26 @@ limitations under the License.
           </b-button-group>
         </template>
         <template v-slot:cell(totalPoints)="data">
-          <div class="row" :data-cy="`usr_progress-${data.item.userId}`">
-            <div class="col-auto">
-              <span class="font-weight-bold text-primary" data-cy="progressPercent">{{ calcPercent(data.value) }}%</span>
+          <div :data-cy="`usr_progress-${data.item.userId}`">
+            <div class="row">
+              <div class="col-auto">
+                <span class="font-weight-bold text-primary" data-cy="progressPercent">{{ calcPercent(data.value) }}%</span>
```
Comment (mosefer) [inline dashboard/src/components/users/UsersTable.vue:66]: span showing percent could use a better aria-label for screenreader users

File: dashboard/src/components/users/UsersTable.vue [hunk 3]
```diff
@@ -60,17 +60,26 @@ limitations under the License.
...
+              </div>
             </div>
-            <div class="col text-right">
-              <span class="text-primary font-weight-bold" data-cy="progressCurrentPoints">{{ data.value | number }}</span> / <span class="font-italic" data-cy="progressTotalPoints">{{ totalPoints | number }}</span>
+            <b-progress :max="totalPoints" class="mb-3" height="5px" variant="info">
+              <b-progress-bar :value="data.value"  :aria-label="`Progress for ${data.item.userId} user`"></b-progress-bar>
+            </b-progress>
+            <div v-if="data.item.userMaxLevel || data.item.userMaxLevel === 0" class="row" data-cy="progressLevels">
+              <div class="col">
+                <i class="fas fa-trophy skills-color-levels" /> <span class="font-italic">Current Level: </span>
```
Comment (mosefer) [inline dashboard/src/components/users/UsersTable.vue:77]: icon should have aria-hidden attribute

File: dashboard/src/components/users/UsersTable.vue (hunk 1 above)
Comment (sudo-may) [reply dashboard/src/components/users/UsersTable.vue]: :+1:

File: dashboard/src/components/users/UsersTable.vue (hunk 3 above)
Comment (sudo-may) [reply dashboard/src/components/users/UsersTable.vue]: :+1:

File: dashboard/src/components/users/UsersTable.vue (hunk 2 above)
Comment (sudo-may) [reply dashboard/src/components/users/UsersTable.vue]: :+1:

**Pull request merged**